app/PasswordStorage.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace App;
+
+class PasswordStorage
+{
+    
+    static function set(string $pwd)
+    {
+        if (self::apcuAvailable()) {
+            apcu_store('firefly_fints_bank_password', $pwd);
```

Comment:
Could you add asserts here to the `apcu_store` and `apcu_fetch` calls that give meaningfull error messages, when things are not working. I think you would have to check the "success" flag of apcu_fetch and the return value of apcu_store?